cmake/std/PullRequestLinuxGCC4.8.4TestingSettings.cmake

```diff
@@ -22,6 +22,9 @@ set(Tpetra_INST_SERIAL ON CACHE BOOL "Set by default for PR testing")
 # Disable just one Teko sub-unit test that fails with openmpi 1.10 (#2712)
 set (Teko_DISABLE_LSCSTABALIZED_TPETRA_ALPAH_INV_D ON CACHE BOOL "Temporarily disabled in PR testing")
 
+# Enable the ensemble reduction in this build to maintain Stokhos with ensemble reduction (#7067)
+set (Stokhos_ENABLE_Ensemble_Reduction ON CACHE BOOL "Enable reduction across ensemble components")
```

Comment:
Do we want to put this change in a PR for one of the more recent GCC versions instead (e.g., 4.9, 7.2 or 8.3)?  I suspect 4.8.4 will go away in the not too distant future.  Comments @trilinos/framework?

Reply:
The default compiler for RHEL 7 is 4.8.5. That is the biggest factor in keeping the build for now. I cannot accurately predict how long the build will be around, but its removal is not currently imminent.

Reply:
OK, for now I think we will just go ahead and add the change to the 4.8.4 PR builds.